services/app-web/src/pages/App/App.tsx

```diff
@@ -35,9 +36,14 @@ function App({
     apolloClient: ApolloClient<NormalizedCacheObject>
     s3Client: S3ClientT
 }): React.ReactElement {
-    useEffect(() => {
-        logEvent('on_load', { success: true })
-    }, [])
+
+    // This is a hacky way to fake feature flags before we have feature flags.
+    // please avoid reading env vars outside of index.tsx in general.
+    const environmentName = process.env.REACT_APP_STAGE_NAME || ''
+    const isProdEnvironment = ['prod', 'val'].includes(environmentName)
+    const jiraTicketCollectorURL = `https://meghantest.atlassian.net/s/d41d8cd98f00b204e9800998ecf8427e-T/-9zew5j/b/7/c95134bc67d3a521bb3f4331beb9b804/_/download/batch/com.atlassian.jira.collector.plugin.jira-issue-collector-plugin:issuecollector/com.atlassian.jira.collector.plugin.jira-issue-collector-plugin:issuecollector.js?locale=en-US&collectorId=e59b8faf`
+    
+    useScript(jiraTicketCollectorURL, !isProdEnvironment)
```

Comment:
Could we wrap the useScript call inside `if (!isProdEnvironment)` or does it need to be called every time?

Reply:
According to [rule of hooks](https://reactjs.org/docs/hooks-rules.html#explanation) can't conditionally run a hook